test/test_id_length.py

```diff
@@ -10,16 +10,21 @@ def check_id_length(feature_name):
         df = release.gtf.dataframe(contig="Y")
         assert feature_name in df, \
             "%s not a column in DataFrame for %s" % (feature_name, release)
+
+
         # Ensembl IDs are formatted like ENSG00000223972
         # which is always length 15
-        ids = df[feature_name]
+        ids = df[feature_name].dropna()
```

Comment:
The `dropna()` moved from the lengths (which is confusing...NaN's propagate through `series.str.len()`) to the ID field itself.